lib/delayed/backend/sqs.rb

```diff
@@ -16,13 +16,41 @@ class Job
         field :last_error,  :type => String
         field :queue,       :type => String
 
+        def self.buffering?
+          @buffering
+        end
+
+        def buffering?
+          self.class.buffering?
+        end
+
+        def self.start_buffering!
+          @buffering = true
+        end
+
+        def self.stop_buffering!
+          @buffering = false
+        end
+
+        def self.clear_buffer!
+          @buffer = nil
+        end
+
+        def self.buffer
+          @buffer ||= {}
+        end
```

Comment:
IMO those `self.xxxxx` methods should be grouped within a `class << self` block.